packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -40,6 +41,54 @@ flutter:
   assets:
     - assets/images/
 
+  fonts:
```

Comment:
Are these fonts really necessary?

Reply:
If we don't specify any fonts, it just uses the regular Ahem block font for testing, since the example app uses WorkSans and the tests only have access to Roboto and .SF UI unless no additional fonts are included in the project or packages. 

I've spent a few hours trying to look for a more elegant solution, but didn't have much success. I can still look into it, or at least reducing the amount of fonts we have right now, since we probably don't need everything

Reply:
Unneeded fonts have been removed